src/btree/col_srch.c

```diff
@@ -328,8 +339,12 @@ __wt_col_search(
             cbt->compare = 0;
         else if (recno < cbt->recno)
             cbt->compare = 1;
-        else
+        else {
```

Comment:
We can implement this as an else if case here?

Reply:
I could add a `||` in the statement above that leads to setting `cbt->compare = 1` but I have added a new `else if` statement in e0d9ed857124fc739c4ec42078854ae04afc4251, let me know what you think.